test/IDE/complete_rdar147789214.swift

```diff
@@ -0,0 +1,11 @@
+// RUN: %batch-code-completion
+
+// rdar://147789214 - Make sure we insert generic parameters for underlying type.
+
+struct S<T> {}
+typealias Foo = S
+typealias Invalid = (S, S)
```

Comment:
The underlying type here should just be an ErrorType

Reply:
Indeed